core/src/main/java/jenkins/org/apache/commons/validator/routines/DomainValidator.java

```diff
@@ -2002,6 +2002,9 @@ public static synchronized void updateTLDOverride(DomainValidator.ArrayType tabl
      * @return converted input, or original input if conversion fails
      */
     // Needed by UrlValidator
+    //[PATCH]
```

Comment:
WAT? Seems the patch was moved from somewhere

Reply:
The patch is to make the field `public`. (Awkward because the modifier is on a separate line to make it clear what is a patch.)

Reply:
The patches are marked because the whole class has been copied from an Apache commons lib, and so all local modifications have been surrounded with `[PATCH]` for clarity.

Reply:
Short story: I thought it might be nice to add validation in #3082 and Wadeck wanted to use a library with robust validation, but we didn't want to add a new dependency to core and only needed part of the lib. Relevant comments: https://github.com/jenkinsci/jenkins/pull/3082#discussion_r179103785.